mpsl/doc/cx.rst

```diff
@@ -20,11 +20,26 @@ Limitations
 
 This experimental feature has several limitations:
 
-* It supports only the nRF52 SoC family.
 * It does not use radio operation priorities, as the radio protocol requests each operation with a priority value of ``1``.
 * The PTA can deny only TX operations.
   Denying RX or idle listening has no effect.
-* Aborting a TX operation takes a few dozens of microseconds.
+* Aborting a TX operation after the access to medium is revoked takes a few dozens of microseconds.
+
+Principle of operation
+*************
+
+Each radio protocol using the radio coexistence feature (e.g. IEEE 802.15.4) can use a transceiver to perform a given radio operation only when the PTA allows it.
```

Comment:
It seems to me that this section is so generic it fails to provide any valuable information in the end. Suppose that I'm a developer reading this sentence. I might think that I don't want my radio protocol I'm developing to use the radio coexistence feature because then I'm limited to what the PTA allows and it's inconvenient. In my opinion we should avoid very generic and vague language. Look at one of the next sentences:

> The radio protocol implementation must inform the radio coexistence module about which radio operations it wants to perform at that moment or shortly after

What are you supposed to do with this information if you're a developer? Who is "radio coexistence module"? How do you "inform" it about anything? When is "shortly after", 10us, 500us or 100ms? There are more unanswered questions now than before reading the sentence.

I would suggest to rework this entire section to describe the intended usage and behavior of this module in a more down-to-earth way. An example what kind of information I'd expect here:
```
The Radio Coexistence feature allows short-range radio protocols, such as IEEE 802.15.4 or Bluetooth LE,
to communicate with the PTA using `mpsl_cx` API. To perform any radio operation, the protocol drivers
must first request it from the arbiter. The radio operation can be executed only when the PTA grants it.
If the PTA revokes access to the medium when a previously granted operation is ongoing, the radio
protocol must abort it immediately. ...
...
The details of how to request access to the medium from a given PTA or how a given PTA grants this access
depend on the implementation of the PTA. They are transparent to the user of `mpsl_cx` API. ...
```
That's only a suggestion ofc.